internal/authorizedapp/database/authorized_app.go

```diff
@@ -47,12 +47,12 @@ func (aa *AuthorizedAppDB) InsertAuthorizedApp(ctx context.Context, m *model.Aut
 		result, err := tx.Exec(ctx, `
 			INSERT INTO
 				AuthorizedApp
-			  (app_package_name, platform, allowed_regions,
+			  (app_package_name, platform, allowed_regions, health_authority_ids,
```

Comment:
Unrelated, but do we have a long-term strategy for safer database migrations? Right now the app always has to be in complete sync.

Reply:
To do that, we would have to start cutting releases on the repo and provide instructions about code + DB migration order.